src/db.c

```diff
@@ -2851,3 +2837,12 @@ int bitfieldGetKeys(struct serverCommand *cmd, robj **argv, int argc, getKeysRes
     }
     return 1;
 }
+
+int dbHasNoKeys(void) {
```

Comment:
Valkey already agree to use bool.  Reference: https://github.com/orgs/valkey-io/discussions/1278
Thus for this kind of function, bool is much eaiser to understand than an integer return type.

Reply:
fixed https://github.com/valkey-io/valkey/pull/1671/commits/362b6596f92e06215bfac9dc04d6c02346ae2ac7